paint-timing/fcp-only/fcp-background-size.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to background size</title>
+<style>
+    #main {
+        position: relative;
+        width: 100px;
+        height: 100px;
+        background-image: url(../resources/circles.png);
+        background-size: 0 0;
+    }
+
+    #main.contentful {
+        background-size: 100% 100%;
+    }
+
+    #log {
```

Comment:
Can we add this in the test_fcp() script instead of adding it in all tests separately?

Reply:
OK